.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: elixir
+sudo: required
+
+elixir:
+  - 1.7.3
```

Comment:
I believe if you say `1.7` it will pull the latest patch version, generally a good thing IMO.

Are there other Elixir versions you wish to support @boydm?  We should test under all supported versions.  Can be another PR; not to hold this one up.

Reply:
@harlantwood @boydm yeah the versions of Elixir and Erlang are up to you @boydm so let me know and I will update the setup